src/index.css

```diff
@@ -1,4 +1,210 @@
+/* stylelint-disable no-descending-specificity */
```

Comment:
- [ ]  For best practice, kindly make sure you fix your linters errors instead of dasabling them. Please make sure you apply this approach in all your files

Reply:
@tresorsawasawa Thanks for the review. I know how important eslint is, but I disabled it because the linter errors only appeared on GitHub and not on my vscode. I didn't know how else to fix it on GitHub since I had already fixed it in my local repo.(That is, my app ran locally without errors of any sort)